src/test/java/com/wikia/webdriver/common/core/imageutilities/CustomCoordsProvider.java

```diff
@@ -0,0 +1,45 @@
+package com.wikia.webdriver.common.core.imageutilities;
+
+import com.wikia.webdriver.common.driverprovider.DriverProvider;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import ru.yandex.qatools.ashot.coordinates.Coords;
+import ru.yandex.qatools.ashot.coordinates.CoordsProvider;
+
+import static java.lang.Math.toIntExact;
+
+public class CustomCoordsProvider extends CoordsProvider {
+
+        private WebDriver driver = DriverProvider.getActiveDriver();
+        private JavascriptExecutor js = (JavascriptExecutor) driver;
+        //TODO:Rewrite it
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)